src/accounting.py

```diff
@@ -383,6 +393,9 @@ def print_money(self, author: AccountId, account: Account, amount: Fraction):
         """Prints `amount` of money on the authority of `author` and deposits it in `account`."""
         account.balance += amount
 
+    def remove_funds(self, author: Account, account: Account, amount: Fraction):
```

Comment:
Isn't this just money printing with negative amounts?

Reply:
yes it was one of the features imade requested